opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -18,26 +18,33 @@
 /**
  * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
-public interface Tracer {
+public interface Tracer extends ActiveSpanSource {
```

Comment:
Maybe helper method like start new span if there is no active span would be helpful.